express/features/support/steps/shared-steps.ts

```diff
@@ -151,3 +152,25 @@ When("they click on the forgot password link in their email", async function ()
     const path = "/create-new-password?userName=registered@gds.gov.uk&confirmationCode=123456";
     await this.goToPath(path);
 });
+
+When(/^they enter their public key:$/, async function (text: string) {
+    await enterTextIntoTextInput(this.page, text, "serviceUserPublicKey");
+});
+
+When("they click element with id {string}", async function (elementId: string) {
+    const toggleKeyViewButton = await getElementById(this.page, elementId);
+    await toggleKeyViewButton.click();
+});
+
+Then("the hidden content of the element with id {string} is displayed", async function (this: TestContext, elementId: string) {
+    const elementAttribute = await this.page.$eval(`#${elementId}`, element => element.hasAttribute("open"));
+    assert.equal(elementAttribute, true);
+});
+
+Then(
+    "the value of the text field with id {string} is {string}",
```

Comment:
```suggestion
    "the value of the text field with the id {string} should be {string}",
```
We are checking this so word the same as other assumptions